sysboundary/sysboundary.cpp

```diff
@@ -432,7 +432,7 @@ bool SysBoundary::classifyCells(dccrg::Dccrg<spatial_cell::SpatialCell,dccrg::Ca
    mpiGrid.update_copies_of_remote_neighbors(SYSBOUNDARIES_NEIGHBORHOOD_ID);
 
    /*Compute distances*/
-   uint maxLayers=3;//max(max(P::xcells_ini, P::ycells_ini), P::zcells_ini);
+   uint maxLayers=22;//max(max(P::xcells_ini, P::ycells_ini), P::zcells_ini);
```

Comment:
This is now a function of the cfg parameter, isn't it? Actually there should be a min of 3, to catch the cases where we'd set all the PML widths to zero.

Reply:
@ykempf . So since maxLayers is defined for every dimension, the pml cannot have different widths per dimension. That's a problem to me. I think in the regular case where solar wind flows in, we would not want the pml applied to the inflow boundary. Unless you set the inflow cells in front of the pml (essentially moving them inwards) but I am not sure this is a good idea.

Reply:
At some point we had discussed this parameter as you were possibly going to use the layer numbering for the PML. Indeed maybe the layers are not needed for PML business, but then here 3 can be put back and the PML magic happens independently of this parameter.